swagger.yaml

```diff
@@ -2094,6 +2094,53 @@ paths:
           schema:
             $ref: '#/definitions/error'
 
+  /liveiso:
+    post:
```

Comment:
We need the following APIs:
1. POST /installer-iso to accept the parameters and upload the ISO to S3
2. GET /installer-iso to return metadata (ssh_public_key <string> and pull_secret_set <boolean>) - we want to show the existing values
3. GET /installer-iso/data - download the ISO
4. GET /installer-iso/presigned - return a presigned URL to download from S3 (the only way to download files in cloud deployment)

We also need:
1. multi-tenancy
2. clean up old images (see how we do it for existing images)

Reply:
Thanks for the review. 

Can you help me understand how and endpoint to return the metadata will be used? UI?

Can you explain more how multi-tenancy should be applied here? Do we need to check for a user id?

I will add presigned url and make sure old images are cleaned up.

Reply:
Summary of our discussion:
1. POST /assisted-service-iso: creates ISO and uploads it to S3 with the user_name in the object path for multi-tenancy
2. GET  /assisted-service-iso/data: downloads the ISO
3. GET /assisted-service-iso/: generates and returns a presigned URL
4. Add ISO cleanup to ExpireObjects()

We can try to get by without a DB table for this iteration, and talk to the UX folks to see if they would need to display the pull secret and SSH key.

Reply:
I have updated the PR to include 1-3. 

What's left is 4. Add ISO cleanup to ExpireObjects is still in progress. Subsystem tests also need updating.

Reply:
I've added the ISO cleanup to imgexpirer.